test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -264,6 +264,19 @@ TEST_F(AtenXlaTensorTest, TestGe) {
   });
 }
 
+TEST_F(AtenXlaTensorTest, TestGeInplace) {
+  at::Tensor a = GetTestTensor({2, 3});
+  at::Tensor b = a.clone();
```

Comment:
Do not simply clone 'a'.
I'd clone, then in one element I'd do -=1.0 and in another one a +=1.0, just to test all combinations.

Reply:
Oh sorry. I meant to do this for the copy of `a` only. Will initialize using `GetTestTensor`. Thank you for the comment.